eng/breakingChanges/breaking-change-doc.ps1

```diff
@@ -0,0 +1,985 @@
+            EstimatedVersion = "Next release"
+        }
+    }
+}
+
+# Function to call LLM API
+function Invoke-LlmApi {
+    param([string]$Prompt, [string]$SystemPrompt = "", [int]$MaxTokens = 3000)
+
+    switch ($Config.LlmProvider) {
+        "github-models" {
+            # Use GitHub CLI with models extension
+            try {
+                $ghArgs = @(
+                    "models", "run",
+                    $Config.LlmModel,
+                    $Prompt,
+                    "--max-tokens", $MaxTokens.ToString(),
+                    "--temperature", "0.1"
+                )
+
+                if ($SystemPrompt) {
+                    $ghArgs += @("--system-prompt", $SystemPrompt)
+                }
+
+                $response = & gh @ghArgs
+                return $response -join "`n"
+            }
+            catch {
+                Write-Error "GitHub Models API call failed: $($_.Exception.Message)"
+                return $null
+            }
+        }
+        default {
+            # Existing API-based providers
+            $headers = @{ 'Content-Type' = 'application/json' }
+            $body = @{}
+            $endpoint = ""
+
+            switch ($Config.LlmProvider) {
+                "openai" {
+                    $endpoint = if ($Config.LlmBaseUrl) { "$($Config.LlmBaseUrl)/chat/completions" } else { "https://api.openai.com/v1/chat/completions" }
+                    $headers['Authorization'] = "Bearer $apiKey"
+
+                    $messages = @()
+                    if ($SystemPrompt) { $messages += @{ role = "system"; content = $SystemPrompt } }
+                    $messages += @{ role = "user"; content = $Prompt }
+
+                    $body = @{
+                        model = $Config.LlmModel
+                        messages = $messages
+                        max_tokens = $MaxTokens
+                        temperature = 0.1
+                    }
+                }
+                "anthropic" {
+                    $endpoint = if ($Config.LlmBaseUrl) { "$($Config.LlmBaseUrl)/messages" } else { "https://api.anthropic.com/v1/messages" }
+                    $headers['x-api-key'] = $apiKey
+                    $headers['anthropic-version'] = "2023-06-01"
+
+                    $fullPrompt = if ($SystemPrompt) { "$SystemPrompt`n`nHuman: $Prompt`n`nAssistant:" } else { "Human: $Prompt`n`nAssistant:" }
+
+                    $body = @{
+                        model = $Config.LlmModel
+                        max_tokens = $MaxTokens
+                        messages = @(@{ role = "user"; content = $fullPrompt })
+                        temperature = 0.1
+                    }
+                }
+            }
+
+            try {
+                $requestJson = $body | ConvertTo-Json -Depth 10
+                $response = Invoke-RestMethod -Uri $endpoint -Method POST -Headers $headers -Body $requestJson
+
+                switch ($Config.LlmProvider) {
```

Comment:
We already have a switch based on LlmProvider above, maybe we should separate REST calls implementations to separate methods per provider (and keep `Invoke-LlmApi` as a single point of entry)